socket-client.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+const io = require("socket.io-client");
+const port = process.env.PORT || 3000;
+const socket = io(`http://localhost:${port}`);
```

Comment:
we will need it to point to localhost for testing but the live request bin otherwise